Babylon/utils/configuration.py

```diff
@@ -1,11 +1,12 @@
 import os
```

Comment:
You are committing something that has already been pushed.
Rebase was not done correctly.